docs/api.md

```diff
@@ -70,6 +70,7 @@ These are the API group versions which are curently available when using Antrea.
 | CRD | CRD version | Introduced in | Deprecated in | Removed in |
 |---|---|---|---|---|
 | `ClusterGroup` | v1alpha2 | v1.0.0 | v1.1.0 | v1.12.0 [^1] |
+| `ClusterGroup` | v1alpha3 | v1.0.0 | v1.13.0 | N/A |
```

Comment:
```suggestion
| `ClusterGroup` | v1alpha3 | v1.0.0 | v1.13.0 | v2.0.0 |
```

Reply:
Just to confirm: are we planning to remove both v1alpha2 and v1alpha3 ClusterGroup in v2.0.0?

Reply:
I think our conclusion is "We should respect our deprecation policy and treat alpha APIs whose feature gate stage are already beta as beta. We may remove deprecated APIs in 2.0 if the policy allows, otherwise defer to 2.1, 2.2, or 2.2 accordingly.".
I didn't see a feature gate for ClusterGroup, I suppose it's part of `AntreaPolicy` and it's already Beta since v1.0. @tnqn should this ClusterGroup API be removed after 9 months after this API is upgraded to Beta?

Reply:
@GraysonWu I feel you can keep the line as it is, I feel we might be unable to remove the ClusterGroup in v2.0.0 if we follow the K8s deprecate policy. We can update the `Removed in` field when it's truly deleted.